C# projection does not recognize `Guid` type declared from C++/WinRT component

**Describe the bug**
Declaring a `Guid` (winrt::guid) type in the C++/WinRT IDL file, to create a WinRT projection, will cause the C# projection to complain about not finding the namespace for it. 

**To Reproduce**
1. In the `idl` file of a C++/WinRT component declare a new method/delegate which passes a `Guid` type argument

Example:
` delegate void MyDelegate(IInspectable parameter, Guid guid);`

2. Try to compile and generate the C# projection and the following error will be displayed:

> C:\src\MyCSProject\obj\x64\Debug\net6.0-windows10.0.19041.0\Generated Files\MyExampleFile.cs(775,123,775,127): error CS0246: The type or namespace name 'Guid' could not be found (are you missing a using directive or an assembly reference?)

**Expected behavior**
I would expect the `Guid` type to be recognized in the C# .NET projection.

**Version Info**

- Microsoft.Windows.CsWinRT: v2.0.1
- Microsoft.Windows.SDK.BuildTools: v10.0.22621.755
- Microsoft.WindowsAppSDK: v1.2.221116.1
- .NET: net6.0-windows10.0.19041.0

--- a/src/Tests/TestComponentCSharp/Class.cpp
+++ b/src/Tests/TestComponentCSharp/Class.cpp
@@ -298,6 +298,18 @@ namespace winrt::TestComponentCSharp::implementation
     {
         _collectionEvent(sender, arg0, arg1);
     }
+    winrt::event_token Class::GuidEvent(TestComponentCSharp::EventWithGuid const& handler)
+    {
+        return _guidEvent.add(handler);
+    }
+    void Class::GuidEvent(winrt::event_token const& token) noexcept
+    {
+        _guidEvent.remove(token);
+    }
+    void Class::InvokeGuidEvent(winrt::guid const& correlationGuid)
+    {
+        _guidEvent(correlationGuid);
+    }
     winrt::event_token Class::NestedEvent(EventHandler<IVector<int32_t>> const& handler)
     {
         return _nestedEvent.add(handler);
@@ -334,6 +346,10 @@ namespace winrt::TestComponentCSharp::implementation
     {
         _returnEvent(arg0);
         return arg0;
+    }
+    winrt::guid Class::TestReturnGuid(winrt::guid const& arg)
+    {
+        return arg;
     }
     int32_t Class::IntProperty()
     {

--- a/src/Tests/TestComponentCSharp/Class.h
+++ b/src/Tests/TestComponentCSharp/Class.h
@@ -20,6 +20,7 @@ namespace winrt::TestComponentCSharp::implementation
         winrt::event<EventHandler2> _event2;
         winrt::event<EventHandler3> _event3;
         winrt::event<EventHandlerCollection> _collectionEvent;
+        winrt::event<EventWithGuid> _guidEvent;
         winrt::event<Windows::Foundation::EventHandler<Windows::Foundation::Collections::IVector<int32_t>>> _nestedEvent;
         winrt::event<Windows::Foundation::TypedEventHandler<TestComponentCSharp::Class, Windows::Foundation::Collections::IVector<hstring>>> _nestedTypedEvent;
         winrt::event<TestComponentCSharp::EventWithReturn> _returnEvent;
@@ -124,6 +125,9 @@ namespace winrt::TestComponentCSharp::implementation
         winrt::event_token CollectionEvent(TestComponentCSharp::EventHandlerCollection const& handler);
         void CollectionEvent(winrt::event_token const& token) noexcept;
         void InvokeCollectionEvent(TestComponentCSharp::Class const& sender, Windows::Foundation::Collections::IVector<int32_t> const& arg0, Windows::Foundation::Collections::IMap<int32_t, hstring> const& arg1);
+        winrt::event_token GuidEvent(TestComponentCSharp::EventWithGuid const& handler);
+        void GuidEvent(winrt::event_token const& token) noexcept;
+        void InvokeGuidEvent(winrt::guid const& correlationGuid);
         winrt::event_token NestedEvent(Windows::Foundation::EventHandler<Windows::Foundation::Collections::IVector<int32_t>> const& handler);
         void NestedEvent(winrt::event_token const& token) noexcept;
         void InvokeNestedEvent(TestComponentCSharp::Class const& sender, Windows::Foundation::Collections::IVector<int32_t> const& arg0);
@@ -133,6 +137,7 @@ namespace winrt::TestComponentCSharp::implementation
         winrt::event_token ReturnEvent(TestComponentCSharp::EventWithReturn const& handler);
         void ReturnEvent(winrt::event_token const& token) noexcept;
         int32_t InvokeReturnEvent(int32_t const& arg0);
+        winrt::guid TestReturnGuid(winrt::guid const& arg);
 
         int32_t IntProperty();
         void IntProperty(int32_t value);

--- a/src/Tests/TestComponentCSharp/TestComponentCSharp.idl
+++ b/src/Tests/TestComponentCSharp/TestComponentCSharp.idl
@@ -12,6 +12,7 @@ namespace TestComponentCSharp
     delegate void EventHandler3(Class sender, Int32 arg0, String arg1);
     delegate void EventHandlerCollection(Class sender, Windows.Foundation.Collections.IVector<Int32> arg0, Windows.Foundation.Collections.IMap<Int32, String> arg1);
     delegate Int32 EventWithReturn(Int32 arg);
+    delegate void EventWithGuid(Guid correlationGuid);
 
     [flags]
     enum FlagValue
@@ -184,13 +185,16 @@ namespace TestComponentCSharp
         event EventHandler3 Event3;
         void InvokeEvent3(Class sender, Int32 arg0, String arg1);
         event EventHandlerCollection CollectionEvent;
+        event EventWithGuid GuidEvent;
         void InvokeCollectionEvent(Class sender, Windows.Foundation.Collections.IVector<Int32> arg0, Windows.Foundation.Collections.IMap<Int32, String> arg1);
+        void InvokeGuidEvent(Guid correlationGuid);
         event Windows.Foundation.EventHandler<Windows.Foundation.Collections.IVector<Int32> > NestedEvent;
         void InvokeNestedEvent(Class sender, Windows.Foundation.Collections.IVector<Int32> arg0);
         event Windows.Foundation.TypedEventHandler<Class, Windows.Foundation.Collections.IVector<String> > NestedTypedEvent;
         void InvokeNestedTypedEvent(Class sender, Windows.Foundation.Collections.IVector<String> arg0);
         event EventWithReturn ReturnEvent;
         Int32 InvokeReturnEvent(Int32 arg0);
+        Guid TestReturnGuid(Guid arg);
 
         Int32 IntProperty;
         event Windows.Foundation.EventHandler<Int32> IntPropertyChanged;

--- a/src/cswinrt/main.cpp
+++ b/src/cswinrt/main.cpp
@@ -343,7 +343,7 @@ Where <spec> is one or more of:
 
             writer eventHelperWriter("WinRT");
             write_file_header(eventHelperWriter);
-            eventHelperWriter.write("namespace WinRT\n{\n%\n}", bind([&](writer& w) {
+            eventHelperWriter.write("using System;\nnamespace WinRT\n{\n%\n}", bind([&](writer& w) {
                 for (auto&& [key, value] : typeNameToEventDefinitionMap)
                 {
                     w.write("%", value);

